bg1npc/bg1npc.tp2

```diff
@@ -22,6 +22,25 @@ README ~bg1npc/readme-bg1npc-%LANGUAGE%.html~ ~bg1npc/readme-bg1npc.html~
 /* ALWAYS = Before each and any component, do this. */
 ALWAYS
 	INCLUDE ~bg1npc/lib/bg1npc_always.tpa~
+
+<<<<<<<< .../fl#inline/newlines
+%LNL%
+>>>>>>>>
+
+  OUTER_SPRINT tra_path ~bg1npc/tra~
+
+  ACTION_DEFINE_ARRAY fl#noconvert BEGIN setup-osx setup-unix setup-win32 END
+  ACTION_DEFINE_ARRAY fl#reload BEGIN bg1npc END
+  LAF HANDLE_CHARSETS

```

Comment:
This function is already invoked by `bg1npc/lib/bg1npc_always.tpa` script so please extend this one instead of calling the function twice.

Reply:
OK, I've withdrawn this change of the bg1npc.tp2 file, though I'm not so sure about the bg1npc_always.tpa change. 
But no matter what changes are made, the console  during installation or the weidu.log file after install always be garbled. So just use the English setup.tra for now.

Reply:
> This function is already invoked by `bg1npc/lib/bg1npc_always.tpa` script so please extend this one instead of calling the function twice.

My fault. have been unchanged.
But now, the console during reinstallation or the weidu.log file after install always be garbled. how to fix it?

Reply:
> This function is already invoked by `bg1npc/lib/bg1npc_always.tpa` script so please extend this one instead of calling the function twice.

My fault. have been unchanged.

And now it test passed. everything looks fine.